linkis-engineconn-plugins/spark/src/main/java/org/apache/linkis/engineplugin/spark/datacalc/service/strategy/ClickHouseStrategy.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.linkis.engineplugin.spark.datacalc.service.strategy;
+
+public class ClickHouseStrategy extends NormalStrategy {
+
+  @Override
+  public String defaultDriver() {
+    return "ru.yandex.clickhouse.ClickHouseDriver";
+    // return "com.clickhouse.jdbc.ClickHouseDriver";  // >= 0.3.2
+  }
+
+  @Override
+  public String defaultPort() {
+    return "8123";
```

Comment:
Can configuration be supported?

Reply:
This is clickhouse‘s default port, it will be used if the port is not configured in linkis-datasource.